app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<main class="recipe_food_container">
+  <h3 class="recipe-name"><%= @recipe.name %></h3>
+
+  <section id="recipe_food_details">
+    <div class="description-container">
+      <div class="recipe_detail_container">
+        <p> Preparation Time: <%= @recipe.preparation_time %> hour </p>
+        <p> Cooking Time: <%= @recipe.cooking_time %> hour </p>
+        <p> <%= @recipe.description %> </p>
+      </div>
+    </div>
+    <div>
+      <% if @recipe.user_id == current_user.id %>
+        <%= form_with model: @recipe, url: toggle_recipe_path(@recipe), method: :patch, local: true do |form| %>
+          <% new_public_status = !@recipe.is_public %>
+          <button type="submit" class="toggle-button <%= @recipe.is_public ? 'private' : 'public' %>">
+            <%= new_public_status ? 'status: Private' : 'status: Public' %>
+          </button>
```

Comment:
- [ ] Great job on your capstone project's UI. it's truly impressive! 😊 However, there's a small adjustment I'd recommend to ensure that your design aligns with the wireframe provided in the project requirements.

   - When examining your recipe details page, you'll notice that the button used to switch a recipe between `public` and `private` doesn't match the wireframe's design. In the wireframe, the button is a toggle, more like a switch, but in your current design, it's not. I'd suggest considering this change, as using a toggle button would make it more intuitive for users to understand that they can switch between making a recipe `public` and `private`. 🔄🔒 See below for more details:
   
<table>
<tr>
<th align=“middle”>Wireframe</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1140" src="https://github.com/over-geek/Recipe-app/assets/107192076/2be8b18e-7c98-46c2-a1c3-7e90e8991ccb">
</td>
<td>
<img width="1440" alt="Screenshot 2023-10-20 at 22 15 10" src="https://github.com/over-geek/Recipe-app/assets/107192076/75df2fea-a2f6-4f72-8812-8b81a751c5c0">
</td>
</tr>
</table>

This adjustment should not only enhance the visual consistency with the project requirements but also improve the overall user experience. 

Happy coding! 😊👨‍💻🚀